plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -1024,15 +1023,23 @@ function upgrade(packageFolder, skipPackageUpdates, headlampPluginVersion) {
    */
   function runCmd(cmd, folder) {
     console.log(`Running cmd:"${cmd}" inside of ${folder} abs: "${resolve(folder)}"`);
-    try {
-      child_process.execSync(cmd, {
-        stdio: 'inherit',
-        encoding: 'utf8',
-      });
-    } catch (e) {
+    const result = child_process.spawnSync(cmd, {
```

Comment:
Curious, why is this one not execFileSync? (assuming you can split cmd?)
Not saying it's better, just curious.